hypothesis-python/src/hypothesis/core.py

```diff
@@ -667,7 +668,8 @@ def run(self):
                         "To get the future behaviour, update your settings to "
                         "include database=None."
                     )
-                    % (self.test.__name__,)
+                    % (self.test.__name__,),
+                    since="2018-09-01",
```

Comment:
This deprecation was introduced on `2017-11-30`, from fc74464 in #1000.

(The error message was later modified in #1557.)